### DISCUSSION TITLE
Welcome to python-discord-windows-controller Discussions!

### DISCUSSION THREAD

**Discussion by mochiron-desu** [EpicBotCoders/python-discord-windows-controller#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dir2alex** [comment]
you are the big gay :^(

**Comment by dir2alex** [comment]
please help on the last step im stuck!!!!!!!!!!!

**Reply by mochiron-desu** [reply]
OKAYYY

**Reply by mochiron-desu** [reply]
I have updated the code try that one
